custom/icds_reports/utils/data_pull/Oct-2019.sql

```diff
@@ -0,0 +1,126 @@
+/*
+# Total AWCs
+# AWCs Launched
+# Districts launched
+# Avg. # of Days AWCs open
+*/
+SELECT
+state_name,
+num_awcs,
+num_launched_awcs,
+num_launched_districts,
+awc_days_open,
+CASE WHEN num_launched_awcs>0 THEN awc_days_open/num_launched_awcs ELSE awc_days_open END AS average_awc_open
+FROM agg_awc_monthly WHERE aggregation_level=1 AND month='2019-10-01'
+
+
+-- # of AWCs that submitted Infra form
+SELECT state_name, sum(num_awc_infra_last_update)
+FROM agg_awc_monthly
+WHERE aggregation_level=1 AND month='2019-10-01'
+GROUP BY state_name
+
+/*
+# of AWCs that reported available drinking water
+# of AWCs that reported unavailable drinking water
+# of AWCs that reported available functional toilet
+# of AWCs that reported unavailable functional toilet
+*/
+SELECT state_name,
+count(*) FILTER (WHERE infra_clean_water IN (1,2,3,5)) AS "Available drinking water",
+count(*) FILTER (WHERE infra_clean_water=0) AS "Unavailable drinking water",
+count(*) FILTER (WHERE infra_functional_toilet=1) AS "Available functional toilet",
+count(*) FILTER (WHERE infra_functional_toilet=0) AS "Unavailable functional toilet"
+FROM agg_awc_monthly
+WHERE aggregation_level=5 AND month='2019-10-01'
+GROUP BY state_name
```

Comment:
when we roll up by summing from awc to state we also avoid counting the test locations
so you can add a where clause like this
`where awc_is_test<>1 and supervisor_is_test<>1 and block_is_test<>1 and district_is_test<>1`

Reply:
okay, thanks.

Reply:
https://github.com/dimagi/commcare-hq/pull/25893/commits/e00aed759956eac3073a7f9d4a07a1dcfa80761e